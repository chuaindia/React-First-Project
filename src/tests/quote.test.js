import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../components/pages/quote';

it('Quotes page will be displayed properly', () => {
  const text = renderer
    .create(<Quote />).toJSON;
  expect(text).toMatchSnapshot();
});
