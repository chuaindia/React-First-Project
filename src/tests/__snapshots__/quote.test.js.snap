// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quotes page will be displayed properly 1`] = `[Function]`;
